packages/manager/modules/pci/src/components/project/flavors-list/flavor-group.scss

```diff
@@ -0,0 +1,7 @@
+.flavors-list {
+  .new-flavor-badge {
+    position: relative;
+    top: -2.5rem;
+    float: right;
+  }
+}
```

Comment:
Question: Cant we add class directly on span ? Why are we adding it from top div ?